cnd/src/config/file.rs

```diff
@@ -45,6 +46,28 @@ pub struct Bitcoin {
     pub fees: Option<BitcoinFees>,
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct BitcoinFees {
+    /// Static value to use
+    #[serde(default)]
+    #[serde(with = "::bitcoin::util::amount::serde::as_sat::opt")]
+    pub sat_per_vbyte: Option<bitcoin::Amount>,
+    /// XOR, use CypherBlock estimation service
+    #[serde(default)]
```

Comment:
Do we need `serde(default)` for `Option`s?

Reply:
Don't we? I'll check.